doc/sample_repos.md

```diff
@@ -191,26 +210,45 @@ RStudio support with JupyterHub*
 ### Files
 ```
 Dockerfile
+How to Launch RStudio.ipynb
```

Comment:
does this file name have slashes for the spaces?

Reply:
ah no - I think instead I'm just going to rename this to `instructions.ipynb`